src/test/java/net/sf/jabref/logic/exporter/MSBibExportFormatTestFiles.java

```diff
@@ -70,15 +74,17 @@ public final void testPerformExport() throws IOException, SaveException {
         String xmlFileName = filename.replace(".bib", ".xml");
         Path importFile = resourceDir.resolve(filename);
         String tempFilename = tempFile.getCanonicalPath();
+
         List<BibEntry> entries = testImporter.importDatabase(importFile, StandardCharsets.UTF_8).getDatabase()
                 .getEntries();
 
         msBibExportFormat.performExport(databaseContext, tempFile.getPath(), charset, entries);
 
-        List<String> expected = Files.readAllLines(resourceDir.resolve(xmlFileName));
-        List<String> exported = Files.readAllLines(Paths.get(tempFilename));
-        Collections.sort(expected);
-        Collections.sort(exported);
-        assertEquals(expected, exported);
+        Builder control = Input.from(Files.newInputStream(resourceDir.resolve(xmlFileName)));
+        Builder test = Input.from(Files.newInputStream(Paths.get(tempFilename)));
+
+        assertThat(test, CompareMatcher.isSimilarTo(control)
```

Comment:
Once upon a time there was a sad test code because we decided against using hamcrest.
Thus could you change the hamcrest-like syntax `assertThat(something readable)` to the good-old `assertEquals`? I think xmlunit also supports these through http://xmlunit.sourceforge.net/api/org/custommonkey/xmlunit/XMLAssert.html